src/main/java/seedu/address/commons/util/VisitUtil.java

```diff
@@ -0,0 +1,27 @@
+package seedu.address.commons.util;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Optional;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.visit.Visit;
+
+/**
+ * Helper class for Visit Todos
+ */
+public class VisitUtil {
+    /**
+     * Get ongoing visit if exists. For Visit Commands that require an ongoing visit.
+     */
+    public static Visit ensureOngoingVisitExistsAndGet(Model model, String messageNoOngoingVisit)
```

Comment:
getOngoingVisitIfExists?

Reply:
I didn't simplify it to that because the function also throws a CommandException. But sounds good to change it to that also. Will change + modify the java method comment.